packages/neuron-wallet/package.json

```diff
@@ -59,7 +59,7 @@
     "reflect-metadata": "0.1.13",
     "rxjs": "6.5.3",
     "sha3": "2.0.7",
-    "sqlite3": "5.0.8",
+    "sqlite3": "5.1.2",
```

Comment:
Is it for updating Node.js?

Reply:
> Is it for updating Node.js?

Yes. It will cause some problems when we execute this command `lerna run rebuild:nativemodules`.